src/backend.rs

```diff
@@ -272,19 +342,19 @@ pub fn prologue(ctx: &mut Context, stack_slots: u32) {
         ; mov rbp, rsp
         ; sub rsp, framesize
     );
-    ctx.sp_depth += aligned_stack_slots - stack_slots;
+    ctx.sp_depth.0 += aligned_stack_slots - stack_slots;
```

Comment:
Now that sp_depth is a struct, can you add functions to it named something like `reserve` (for decrementing) and `free` (for incrementing), so that we don't have `.0` references scattered around?

Reply:
Done!